js-lib/b1.js

```diff
@@ -1,7 +1,7 @@
 import { setBenchmarkResult, gen, N, benchmarkTime, logMemoryUsed, getMemUsed, runBenchmark } from './utils.js'
-import * as prng from 'lib0/prng'
-import * as math from 'lib0/math'
-import * as t from 'lib0/testing'
+import * as prng from 'lib0/prng.js'
```

Comment:
By importing this without the `.js` extension, the import will be resolved using  "conditional exports" which ensure that this library can be consumed by different module bundlers as well.

Conditional exports are available in the latest release of nodejs v12. But I recommend updating to a current release of nodejs (v16 or v18). 

Can you please revert these changes? I don't support node v12 anymore. I will make that explicit by changing the `engine` field in package.json.

Reply:
If I don't add the `.js` extensions, I get errors such as the following in Node 18.2:

```
Error [ERR_UNSUPPORTED_DIR_IMPORT]: Directory import '/Users/martin/dev/cl/crdt-benchmarks/node_modules/lib0/prng' is not supported resolving ES modules imported from /Users/martin/dev/cl/crdt-benchmarks/js-lib/utils.js
Did you mean to import lib0/prng.js?
```

Is there another way of resolving this error?

Reply:
That is interesting. I just migrated to Node 18.2 and I don't get this error.

Either your node version is not linked correctly, or there is something wrong with Node.

Could you please state your operating system (+ file system if you use something esoteric like ZFS) and also run `node -v` to get the specific version? I'm going to try to reproduce this and report the issue.

Does the same issue happen when you run `cd benchmarks/automerge-wasm && node run.js` to run the benchmarks?

Is there any chance that you have `--experimental-specifier-resolution=node` activated? Maybe as an alias to node?

Reply:
Very strange. I'm running Node v18.2.0, freshly installed today (via nvm) on macOS 12.4 (Intel CPU), and nothing esoteric as far as I know. I tried running it in Docker with the same version of Node and didn't get the error there, so possibly it's OS-specific. Anyway, it seems that if I add the `--experimental-specifier-resolution=node` option the error goes away, and it still seems to work in Docker, so perhaps that is the best solution for now.

I've updated the PR to revert the changes to the imports, and add the `--experimental-specifier-resolution=node` option to the `npm start` script.